packages/theming/theme-types/src/Color.types.ts

```diff
@@ -439,6 +439,10 @@ export interface ControlColorTokens {
   buttonDisabledSecondaryContent: ColorValue;
   buttonHoveredSecondaryContent: ColorValue;
   buttonPressedSecondaryContent: ColorValue;
+
+  checkboxBackground: ColorValue;
+  checkboxBackgroundDisabled: ColorValue;
+  checkmarkColor: ColorValue;
```

Comment:
Do these names match the fluentui-token-pipeline names? Also FYI @rurikoaraki since she's working on that

Reply:
Ohh didn't check that :/

Reply:
It would be nice if they match up so it's easier to integrate but if they aren't in the pipeline yet, then that's fine!